build/scripts/darwin_amd64.sh

```diff
@@ -13,7 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eu
+set -eux
+
+CURRENT_TAG=$(git tag --sort v:refname | tail -n 1)
```

Comment:
so "git" command may not be there in all build environments (ex. kokoro Windows). Also, how does this tag value relate to the Version=dev you introduced to main.go? From the top of my head, would it make sense to store a manually updated "version.txt" so it it can be supplied to the ldflags as well as return programmatically?

Reply:
Perhaps in Kokoro environments it instead makes sense to use the information found in the `METADATA` file? 

In those cases we can add a check to see if this variable is already defined

Reply:
@andyrzhao Friendly ping :D